server/graphql/asset_catalogue/Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository = { path = "../../repository" }
 service = { path = "../../service" }
 util = { path = "../../util" }
 graphql_core = { path = "../core" }
+graphql_types = { path = "../types" }

```

Comment:
Sad to add a build dependancy, but didn't want to duplicate DeleteReponseType 😢  
That should maybe be moved to core but already more than enough changes in this PR.

Reply:
Good point - moving to core would be better.. but not now!